lib/puma/dsl.rb

```diff
@@ -865,5 +865,13 @@ def nakayoshi_fork(enabled=true)
     def max_fast_inline(num_of_requests)
       @options[:max_fast_inline] = Float(num_of_requests)
     end
+
+    # Change the backend for the IO selector.
+    #
+    # The default is +:auto+ which will let nio4r choose the backend.
+    #
```

Comment:
Let's mention here that the option is passed directly to nio4r, and that valid options are here: https://github.com/socketry/nio4r/blob/4111b1cee87ce8857558900b8df76a5bd6c3e985/lib/nio/selector.rb#L27

Reply:
WIll do! :+1: